cmd/vulndash/Makefile

```diff
@@ -13,13 +13,14 @@
 # limitations under the License.
 
 # include the common image-building Makefiles
-include $(CURDIR)/../../Makefile.common-image $(CURDIR)/../Makefile.artifact-promoter
+#include $(CURDIR)/../../Makefile.common-image
```

Comment:
is this meant to be left as a comment?

Reply:
@hasheddan -- yep, planning to try something out in the next iteration!